### PR TITLE
feat: improvements and fixes on places and events panel

### PR DESCRIPTION
**Pull request by lorux0**
## What does this PR change?

- fix http request missing auth-chain for "set as interested" on an event 
- empty state on events tab
- add links to urls in the places & events descriptions
- fix http request for getting events of Genesis Plaza (more than parcel 255 parameters)
- disable/enable map zoom when the pointer enters/exits the side panel area
- make a clean search after the map is re-opened

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/Navmap/PlacesAndEventsPanelController.cs [hunk 1]
```diff
@@ -22,18 +23,23 @@ public PlacesAndEventsPanelController(
...
             this.eventInfoPanelController = eventInfoPanelController;
+            this.navmapZoomController = navmapZoomController;
 
             view.CollapseButton.gameObject.SetActive(true);
             view.ExpandButton.gameObject.SetActive(false);
             view.CollapseButton.onClick.AddListener(Collapse);
             view.ExpandButton.onClick.AddListener(Expand);
+
+            view.PointerEnter += DisableMapZoom;
+            view.PointerExit += EnableMapZoom;
```
Comment (davidejensen) [inline Explorer/Assets/DCL/Navmap/PlacesAndEventsPanelController.cs:42]: can you unsub in the dispose?

File: Explorer/Assets/DCL/Navmap/PlacesAndEventsPanelController.cs [hunk 2]
```diff
@@ -42,7 +48,11 @@ public void Show()
             searchResultController.Show();
 
             searchPlacesCancellationToken = searchPlacesCancellationToken.SafeRestart();
-            searchBarController.SearchAndShowAsync(searchPlacesCancellationToken.Token).Forget();
+
+            searchBarController.SearchAndShowAsync("",
```
Comment (davidejensen) [inline Explorer/Assets/DCL/Navmap/PlacesAndEventsPanelController.cs:52]: use string.Empty instead of ""

**Review by davidejensen — APPROVED** [review]
Approved with two minor code suggestion

**Review by dalkia — APPROVED** [review]

File: Explorer/Assets/DCL/Navmap/PlacesAndEventsPanelController.cs (hunk 1 above)
Comment (lorux0) [reply Explorer/Assets/DCL/Navmap/PlacesAndEventsPanelController.cs:42]: Do you find it necessary? This controller is just created once during the whole lifecycle of the app and i dont see that is going to change

**Pull request merged**